packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -51,18 +54,24 @@ const onTxError = (error: any) => {
   }
 }
 
-const isAllowedL2 = async (
-  arbTokenBridge: ArbTokenBridge,
-  l1TokenAddress: string,
-  l2TokenAddress: string,
-  walletAddress: string,
-  amountNeeded: BigNumber,
+const isAllowedL2 = async ({
```

Comment:
For a future PR: let's refactor this into a `getL2TokenAllowance`, similar to what we have for L1 token

Reply:
Referenced in a new issue : https://github.com/OffchainLabs/arbitrum-token-bridge/issues/866